src/python/zquantum/core/interfaces/backend.py

```diff
@@ -120,14 +143,51 @@ def __init__(
         self.device_connectivity = device_connectivity
 
     @abstractmethod
+    def _get_wavefunction_from_native_circuit(
+        self, circuit: Circuit, initial_state=None
+    ) -> Wavefunction:
+        """Get wavefunction from circuit comprising only natively-supported operations.
+
+        Args:
+            circuit: circuit to simulate. Implementers of this function might assume
+              that this circuit comprises only natively-supported operations as decided
+              by self._is_supported predicate.
+            initial_state: amplitudes of the initial state. If `None`, implementers
+              should assume that the initial state is |0...0>, which typically does not
+              require any initialization.
+        Returns:
+            Wavefunction object encoding amplitudes of final state.
+        """
+
+    def is_natively_supported(self, operation: Operation) -> bool:
+        """Determine if given operation is natively supported by this Simulator.
+
+        This method will be used as a predicate in split_circuit function.
```

Comment:
It can be used as a predicate, but I think this is not necessarily the only use, so perhaps docstring should change?

Reply:
Fixed in  51fd75b